### ISSUE TITLE
Train on my own dataset without superpixels

### ISSUE THREAD

**Issue by ron-weiner** [warmspringwinds/pytorch-segmentation-detection#7]
First of all, great work!
I assume that in order to use it I have to train the weights on a dataset with superpixels annotations, or at least some boundry boxes.
Am I right?
What do you suggest me to do if now I have lots of clean videos which I want to make segmentation on some features that are moving in the video?
Should I start with simple boundary boxes classification and only than proceed? or that there is any shortcut for that?

Thanks

**Comment by warmspringwinds** [comment]
Hi @ron-weiner 

You can try this software to create training data:
https://github.com/wkentaro/labelme

and then train using our library 

Let me know if you have any further questions :)

**Comment by ron-weiner** [comment]
Thank you @warmspringwinds , it is a great tool!
Unfortunately I need to do it on  a huge amount of data and the only option that I see is to get bunch of people to label it all.
If i wish make segmentation on moving objects in a video? Do you have another lead besides that?
I started using "https://www.learnopencv.com/object-tracking-using-opencv-cpp-python/" but  I wish to have a great segmentation such as yours

**Comment by warmspringwinds** [comment]
@ron-weiner I will have a look if such tool exists :)

Or if you find something better by that time, I would also be interested in the 
way you approached your problem.

Thank you.

**Comment by ron-weiner** [comment]
For sure, I will let you know!
Thank you!
